data/tools/wmlunits

```diff
@@ -317,23 +317,23 @@ def list_contents():
             sys.stdout.write("ok\n")
         except wmlparser3.WMLError as e:
             ef = open(logname, "w")
-            ef.write("<PARSE ERROR>\n")
+            ef.write("<PARSE_ERROR>\n")
```

Comment:
I don't get the point of adding underscores here. It's just a log to be read by people, right? It's not like it's an actual XML file.

Reply:
Well, it is an HTML file. ``<PARSE ERROR>`` is not illegal _per se_, but ``</PARSE ERROR>`` is.

Reply:
This isn't valid as HTML either, so I'd say changing it from one invalid thing to another invalid thing doesn't make much sense…

Reply:
That's a good question, actually. The procedure that generates the actual ``.html`` file elsewhere expected the closing tags to be their own line. Took care of that.

Reply:
So basically it's passing that fake XML-style tag through something that converts it to HTML?

Reply:
I feel it should be something like `<p class="...">` but I guess that's out of scope here.

Reply:
Moved to #7581